include/itkAttenuationImageFilter.hxx

```diff
@@ -122,6 +131,28 @@ AttenuationImageFilter<TInputImage, TOutputImage, TMaskImage>::BeforeThreadedGen
   m_OutputMaskImage->SetRegions(inputMaskImage->GetLargestPossibleRegion());
   m_OutputMaskImage->Allocate();
   m_OutputMaskImage->FillBuffer(0.0f);
+
+  // Initialize distance weights
+  unsigned fourSigma = std::max(m_FixedEstimationDepth, 16u);
+  m_DistanceWeights.resize(fourSigma);
+  float twoSigmaSquared = fourSigma * fourSigma / 8.0f;
+  for (unsigned i = 0; i < fourSigma; ++i)
+  {
+    m_DistanceWeights[i] = 1.0f - std::exp(i * i / -twoSigmaSquared);
```

Comment:
Would it be reasonable to use an existing mechanism such `itk::GaussianOperator` instead? Advantages would be 1) functionality is explicitly covered in testing and 2) would allow the filter to more easily expose weighting parameters per a common interface.

Reply:
I just took a look at [itk::GaussianOperator](https://itk.org/Doxygen/html/classitk_1_1GaussianOperator.html). It is too complicated for this purpose. If you disagree, you could change this in a follow-up PR.

Reply:
Understood, no problem.